dGame/dComponents/SimplePhysicsComponent.h

```diff
@@ -14,16 +14,24 @@
 
 class Entity;
 
+enum eClimbableType : int32_t {
```

Comment:
Would prefer an enum class here. @EmosewaMC opinion? What has been the conversion recently?

Reply:
I'd agree with @Wincent01 that this should be an enum class to protect against any possible bad comparisons.  
See [this](https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum) stack overflow thread which explains why to prefer classes over plain enums.

Reply:
Neat, did not know that! Will change accordingly